packages/rocketchat-api/server/default/info.js

```diff
@@ -1,5 +1,17 @@
 RocketChat.API.default.addRoute('info', { authRequired: false }, {
 	get: function() {
-		return RocketChat.Info;
+		const user = this.getLoggedInUser();
+
+		if (user && RocketChat.authz.hasRole(user._id, 'admin')) {
+			return {
```

Comment:
This is why your tests are failing, all of the successful results inside the api need to be returned with `RocketChat.API.v1.success` see the return below